addon/components/paper-select-menu-trigger.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import BasicTrigger from 'ember-basic-dropdown/components/basic-dropdown/trigger';
+
+const { computed } = Ember;
+
+export default BasicTrigger.extend({
+  tagName: 'md-select',
+  attributeBindings: [
+    'role',
+    'tabIndex:tabindex',
+    'dropdownId:aria-controls',
+    'ariaLabel:aria-label',
+    'ariaLabelledBy:aria-labelledby',
+    'ariaDescribedBy:aria-describedby',
+    'aria-disabled',
+    'aria-expanded',
+    'aria-haspopup',
+    'aria-invalid',
+    'aria-pressed',
+    'aria-required',
+    'disabled:disabled'
```

Comment:
`attributeBindings` is a concatenated property. This means that it will be concatenated when we `extend` the base class.
We just need to specify `attributeBindings: [ 'disabledAttr:disabled']` in this case, with the following CP:
```js
disabledAttr: computed('disabled', function() {
  return this.get('disabled') ? 'disabled' : null;
})
```